contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -1783,7 +1783,11 @@ pre_transform_target_entry(ResTarget *res, ParseState *pstate,
 					 * for the last_dot position
 					 */
 					if (*colname_start == '-' && *(colname_start+1) == '-')
+					{
```

Comment:
Should we also check for open square brackets = 0 and double quotes being even ?

Reply:
Or could we move this check to this [line](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/666f2526f2f39e0caea1ce232e04d2865c01c830/contrib/babelfishpg_tsql/src/hooks.c#L1816)

Reply:
O , sorry, I didn't see this before merging. Let me check.